tools/torchci/update_test_times.py

```diff
@@ -18,15 +13,15 @@
 TEST_TIME_PER_CLASS_PERIODIC_JOBS_QUERY_NAME = "test_time_per_class_periodic_jobs"
 
 
-def get_file_data_from_rockset():
-    return get_data_from_rockset(file_mode=True)
+def get_file_data_from_clickhouse():
```

Comment:
Nit (so that one does not have to rename next time we migrate :P )
```suggestion
def get_file_data_from_db():
```